### ISSUE TITLE
Issue on docs

### ISSUE THREAD

**Issue by sannaha** [apache/doris-website#1226]
Path:/zh-CN/docs/3.0/sql-manual/sql-functions/table-valued-functions/hdfs

> Related parameters for accessing hdfs:
> ...
> `hadoop.username`:  (required) Can be any string, but cannot be empty.
> ...
> `hadoop.username`: (optional)

How can the same parameter `hadoop.username`  be both required and optional :)

**Comment by KassieZ** [comment]
Good catch...thanks for the feedback, will double check and fix it ASAP
